ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/dropdown.html

```diff
@@ -40,6 +40,7 @@
             readonly="${dropdown.readOnly}"
             required="${dropdown.required}"
             data-sly-attribute.multiple="${dropdown.isMultiSelect ? 'multiple' : ''}">
+        <option id="emptyValue" role="none" style="display: none;"></option>
```

Comment:
is there a reason why emptyValue is needed here?
there is one more option for placeholder. can they both be combined to single option?

Reply:
@nit23uec 
When dropdown (single select) is initialised it always shows the first option as selected. Even though the value is not set in the model. This empty value is useful there.  
Setting selectedIndex = -1 clears the dropdown and we don't require this emptyValue option for clearing now like you. pointed out.

Reply:
in case there is a placeholder, we would get 2 empty options.

Reply:
emptyValue option is not required in the dom when placeholder is not null. can we put that check also.